src/mlpack/bindings/cli/print_help.cpp

```diff
@@ -17,18 +17,16 @@
 
 namespace mlpack {
 namespace bindings {
-namespace cli {
+namespace cmd {
 
 /* Prints the descriptions of the current hierarchy. */
 void PrintHelp(const std::string& param)
 {
   std::string usedParam = param;
-  const std::map<std::string, util::ParamData>& parameters =
-      CLI::Parameters();
-  const std::map<char, std::string>& aliases = CLI::Aliases();
-
-  std::map<std::string, util::ParamData>::const_iterator iter;
-  const util::ProgramDoc& docs = *CLI::GetSingleton().doc;
+  std::map<std::string, util::ParamData>& parameters =
+      IO::Parameters();
```

Comment:
```suggestion
  std::map<std::string, util::ParamData>& parameters = IO::Parameters();
```
This fits on one line too; maybe if you haven't already done it it would be good to set your editor's number of columns to 80?  It doesn't hurt to have lines that are too short, but it's probably good to try and maximize the length when we can. :+1: